foundation-ui/mount/Mount.js

```diff
@@ -0,0 +1,108 @@
+import React, {PropTypes} from 'react';
+
+import MountService from './MountService';
+import {CHANGE} from './MountEvent';
+import ReactUtil from '../utils/ReactUtil';
+
+const METHODS_TO_BIND = [
+  'handleMountServiceChange'
+];
+
+/**
+ * A component to use as a mount point to incorporate views/components provided
+ * by packages. It renders with a/components registered to the `MountService`
+ * with a matching `type`.
+ * It provides a property to configure the wrapping component (`wrapper`) as
+ * well as a property to `limit` the number of rendered elements.
+ *
+ * @example
+ *
+ * <Mount type="widget" limit={10} wrapper={MyWidgetsWrapper}>
+ *     <span className="help-text"></span>
+ * </Mount>
+ *
+ */
+class Mount extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    // Get components and init state
+    this.state = {
+      components: MountService.findComponentsWithType(this.props.type)
+    };
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  componentWillMount() {
+    MountService.addListener(CHANGE, this.handleMountServiceChange);
```

Comment:
Dang, I actually had a look to make sure that I'm using the correct naming scheme but I must have the one file with a bad example... Going to adjust the naming.